torch/_tensor_docs.py

```diff
@@ -2407,6 +2419,18 @@ def callable(a, b) -> number
 
 """)
 
+add_docstr_all('view_as',
+               r"""
+view_as(other) -> Tensor
+
+View this tensor as the same size as :attr:`other`.
```

Comment:
Can you add a sentence that links to `` :meth:`torch.Tensor.view` `` on the constraints of `view`?